packages/core/src/__tests__/ceramic-anchor.test.ts

```diff
@@ -314,30 +318,47 @@ describe('Ceramic anchoring', () => {
     await ceramic2.close()
   })
 
-  it('test the same state anchored twice, first one wins', async () => {
```

Comment:
I still think the original version of this test is useful to have and don't see why we are removing it